src/pages/index.js

```diff
@@ -1,15 +1,36 @@
 import {Helmet} from 'react-helmet';
 
+import EventCard from '../components/EventCard/EventCard';
 import Layout from '../components/Layout';
 
 export default function HomePage() {
+	const dummyEvent = {
```

Comment:
You can export the dummy event to an extra file and then import it back here, so your code looks cleaner.

Reply:
good tip, thank you